Content/WorldGeneration/WorldgenWorld.cs

```diff
@@ -19,6 +19,8 @@ public override void ModifyWorldGenTasks(List<GenPass> tasks, ref double totalWe
 			if (ShiniesIndex != -1)
 			{
 				tasks.Insert(DesertIndex + 1, new PassLegacy("Starlight River Temples", UndergroundTempleGen));
+				tasks.Insert(DesertIndex + 4, new PassLegacy("Starlight River Vitric Desert", VitricGen));
```

Comment:
Why are we generating the desert twice?

Reply:
> Why are we generating the desert twice?

I forgot to remove that line. I'll make that change now.